docker-compose.yml

```diff
@@ -274,6 +274,7 @@ services:
             - BRIDGE_AVG_PARTICIPANT_STRESS
             - BRIDGE_STRESS_THRESHOLD
             - ENABLE_AUTH
+            - JICOFO_ENABLE_AUTH
```

Comment:
Please put this one alphabetically sorted.

Reply:
Done! :)